lib/process/process.go

```diff
@@ -540,6 +545,48 @@ func (p *Process) startAutoscale(ctx context.Context) error {
 	return nil
 }
 
+// startApplicationsSynchronizer starts a service that periodically exports
+// Docker images of the cluster's application images to the local Docker
+// registry.
+//
+// TODO There may be a lot of apps, may be worth parallelizing this.
+func (p *Process) startApplicationsSynchronizer(ctx context.Context) error {
```

Comment:
It should be same as now b/c we already have pretty much the same goroutine that synchronizes bundles (cluster images). I agree though we might need to revisit this synchronization "algo" cause it doesn't scale well. IIRC we added this because the registry was occasionally mysteriously "losing" images, maybe we don't need this constant sync at all now (and just have a command for force-sync if needed).